core/templates/dev/head/pages/base.html

```diff
@@ -119,6 +119,10 @@
       {% endif %}
     </script>
 
+    <!-- Note to developers: This script needs to be loaded before external
```

Comment:
Oh, I see. Hm.

Yeah, I think what you propose is fine. Just add a comment at the top of the karma script saying something like 'this should be kept in parity with how CONSTANTS is set in base.html', and add a comment here saying 'If you modify this, please change [path to karma-constants.js] accordingly'.

(I considered using the same script for both, as you've done now, but it seems to me like we ought to go for the efficiency gain in production since this affects all pages.)